Ginger/Ginger/UserControlsLib/ucGridView/ucGrid.xaml.cs

```diff
@@ -196,7 +200,17 @@ private bool FilterGridRows(object obj)
             //Filter by search text            
             if (!string.IsNullOrEmpty(mFilterSearchText))
             {
-                if (TextFilter(obj, mFilterSearchText) == true)
+                bool matchedSearchText;
+                if (TextFilter != null)
+                {
+                    matchedSearchText = TextFilter.Invoke(obj, mFilterSearchText);
+                }
+                else
+                {
+                    matchedSearchText = DefaultTextFilter(obj, mFilterSearchText);
+                }
+
+                if (matchedSearchText)
                 {
                     return true;
                 }
```

Comment:
ucGrid class is very old and it is very tightly bound with the rest of the code base. Changing it would be opening pandora's box. So till the day it becomes an absolute necessity, this is what we got.